_source/user-guide/users/single-sign-on/index.md

```diff
@@ -29,10 +30,19 @@ and more secure for your company.
 When you enable SSO on your account,
 you're configuring Logz.io to hand off authentication
 to your identity provider.
-You'll be able expand or restrict a user's access to Logz.io,
+You'll be able to expand or restrict a user's access to Logz.io,
 but not add or remove users from within Logz.io.
 
-All authenticated users will have access to your account, unless you configure groups.
+All authenticated users will have access to your account, and existing users will retain the permission levels they had before SSO was enabled, unless you configure groups.
+
+###### How SSO groups work
+
+Groups allow setting **User** or **Admin** level permissions to users when using SSO.
```

Comment:
line 40 isn't clear enough and should include the WIIFM (what's in it for me) - advantage of using **SSO groups** to set access, rather than configuring permissions/privileges for each user individually...


What about Read-only permissions?